helm-chart/values.yaml

```diff
@@ -1,18 +1,30 @@
-# Copyright 2020, NVIDIA CORPORATION.
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Leave the original license

Reply:
Hmm Then should we do PR that only changes the licenses back to the apache?